apps/admin/providers/authProvider.ts

```diff
@@ -26,7 +26,8 @@ export const authProvider = (session?: Session | null): AuthProvider => ({
     console.info("getIdentity", session);
     if (session?.user)
       return Promise.resolve({
```

Comment:
I was thinking that passing the authProvider as the argument of the handlers, does not guarantee the user implements it correctly and might forget to include the `id` for instance... we need only the id right for the log? and that will connect to the admin user? in that case the apiHandler just provides the handler `id` as the third argument

Reply:
we might need other data from getIdentity, let's do that another time.